src/middleware/Auth.js

```diff
@@ -0,0 +1,24 @@
+// third-party libraries
+import jwt from 'jsonwebtoken';
+
+const auth = {
+  /**
+   * @static
+   * @param {object} user
+   * @description Generates token for user
+   * @return {string} string
+   */
+  authenticate(user) {
+    return jwt.sign(
+      {
+        id: user.id,
+      },
+      process.env.SECRET,
```

Comment:
please move this to the `config.js` file

Reply:
Okay
I will look into it immeadiately